tuning/sft_trainer.py

```diff
@@ -344,34 +353,103 @@ def main(**kwargs):  # pylint: disable=unused-argument
         help='Pass a json string representing K:V pairs to be associated\
               to the tuning run in the tracker. e.g. \'{"gpu":"A100-80G"}\'',
     )
-    (
+    return parser
+
+
+def parse_arguments(parser, json_config=None):
+    # accept arguments via command-line or JSON
```

Comment:
Added docstrings for all the methods added. I'm going to leave it in sft_trainer for now since no other files are using the get_parser() or parse_arguments() to keep the pieces together. But I agree if another file ends up needing it, then it should be moved to utils.